//
// Created by stepo on 1/21/18.
//

#ifndef PLATOONING_MESSAGETYPES_HPP
#define PLATOONING_MESSAGETYPES_HPP

#define LV_BROADCAST 0x00000001
#define FV_HEARTBEAT 0x00000002
#define FV_REQUEST 0x00000008
#define LV_ACCEPT 0x00000010
#define LV_REJECT 0x00000020
#define FV_LEAVE 0x00000040
#define REMOTE_CUSTOM 0x01000000
#define REMOTE_CMD 0x02000000
#define REMOTE_LOG 0x04000000

#define REMOTE_PLATOONINGTOGGLE 0x00000100
#define REMOTE_CONTROLTOGGLE 0x00000200
#define REMOTE_CONTROLINPUT 0x00000300
#define REMOTE_USERINTERFACE 0x00000400

#endif //PLATOONING_MESSAGETYPES_HPP
